src/Microsoft.Extensions.Logging.AzureWebAppDiagnostics/AzureWebAppDiagnosticsSettings.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Extensions.Logging.AzureWebAppDiagnostics
+{
+    /// <summary>
+    /// Settings for <see cref="AzureWebAppDiagnosticsLoggerProvider"/>.
+    /// </summary>
+    public class AzureWebAppDiagnosticsSettings
+    {
+        /// <summary>
+        /// A strictly positive value representing the maximum log size in bytes. Once the log is full, no more message will be appended
```

Comment:
The standard message for properties that are get & set starts with "Gets or sets ..."